src/collection-browser.ts

```diff
@@ -423,6 +430,13 @@ export class CollectionBrowser
 
     if (!sortField) return;
     this.sortParam = { field: sortField, direction: this.sortDirection };
+
+    // Lazy-load the alphabet counts for title/creator sort bar as needed
+    if (this.selectedSort === 'title' && !this.titleLetterCounts) {
```

Comment:
Question: for DRY, `fetchTitleLetterCounts()` and `fetchCreatorLetterCounts()` seem remarkably similar. With above line 384 comment, could we simply have 
```
fetchFirstCharCounts();
```
 here? And handle all the logic inside that function, as needed by `this.selectedSort` and the other var(s), including doing nothing?